### ISSUE TITLE
Where are the Ores?

### ISSUE THREAD

**Issue by StephenFlagg** [Beyond-Reality/Beyond-Reality-Divergence#361]
I'm playing Divergence 1.3.4 for my YouTube channel and I can't seem to find most of the (metal) ores.  I can find Iron, Gold, and Copper but nothing else.  I even opened a new world in creative and went searching for hours, but no trace of Nickel, Tin, Aluminum or any of the other ores.
As well, it seems my sluice only collects flint.  I have yet to see any chunks of raw ore; all the ores I can find mine normally.
Are there issues with 1.3.4?

**Comment by Dominance** [comment]
If you can find copper in veins then the others such as bauxite(aluminum) nickel and tin are there. Unless your in spectator mode(gm3) its easier to look for flowers corresponding to their ore types and then below/around that flower. Website is currently down so the charts are as well, how ever you can look through the XML files to find which biome each ores have a increased chance to generate in such as https://github.com/Beyond-Reality/Beyond-Reality-Divergence/blob/master/config/CustomOreGen/modules/custom/BROre.xml#L342

Spending hours looking for something that will last you all game is also kind of the point.
There is a known issue of oregen sometimes messing up on RTG world types, but if you found copper in huge veins then your world should be good

Most of the major game changes come from reasonable realism and you can find all the information here http://reasonable-realism.wikia.com/wiki/Reasonable_Realism_Wiki

Here is the distribution page that has some good information on it http://reasonable-realism.wikia.com/wiki/Ore_Distributions

**Comment by StephenFlagg** [comment]
I'll go back and take a look again.  I did use ATG, but as you said, if I have copper then the rest should be there.  Last night (in creative) I went to Y12 in a Swampland and dug a 3x3 mine for 2000 blocks and didn't see anything but the basic ores.  Also no tiny/sparse ores.

**Comment by Dominance** [comment]
As long as your not seeing vanilla ores stuff should be working. If you are something broke

On Dec 1, 2017 8:54 AM, "StephenFlagg" <notifications@github.com<mailto:notifications@github.com>> wrote:

I'll go back an take a look again. I did use ATG, but as you said, if I have copper then the rest should be there. Last night (in creative) I went to Y12 in a Swampland and dug a 3x3 mine for 2000 blocks and didn't see anything but the basic ores.
